doc/conference_posters.md

```diff
@@ -0,0 +1,9 @@
+# Conference posters
+Here's some useful tools for whne you are desining a new conference poster. 
+
+## Templates and logos
+In the shared google drive we have a [folder with poster templates](https://drive.google.com/drive/folders/1e7-B4K2QuL-Q6hSJn0pEHg_Q7SgRSYad). 
+There you can find powerpoint templates and logos of the main institutions and fundings. 
+
+## Examples of posters from the lab
+In this other [folder](https://drive.google.com/drive/folders/1s5D3lUjc_yjl-tU8ExhMbruQay83ggF8) there are some (old) posters from lab members that can be used as source of inspiration. 
```

Comment:
I would ad a link on how to write the affiliation. A fresh PhD/Postdoc will probably go first through a postern then a paper so better to link affiliation here as well. Chcking how to add links across md files